### DISCUSSION TITLE
How to compute noise in DB, directly

### DISCUSSION THREAD

**Discussion by mshmoon** [Universite-Gustave-Eiffel/NoiseModelling#536] [Q&A] [answered]
When I use the noisemodelling to compute noise , I got HZ value. 
So  I question is how can I get DB value. Thanks .

**Comment by nicolas-f** [comment, marked as answer]
Hi,

How you got that, is it in the database table when using NoiseLevelFromSource in the WPS builder ?

**Reply by mshmoon** [reply]
> Hi,
> 
> How you got that, is it in the database table when using NoiseLevelFromSource in the WPS builder ?

Here are my steps:

1. I loaded buildings.geojson, ground_ type.geojson,  roads.geojosn.  receivers.geojson,  dem.geojson

2. I use the noise_ level_ from_ Traffic function to calculation

3. I use export_ Table function to export LDAY_ GEOM.geojson

4. I check LDAY_ GEOM.geojson, found all value is HZ, no DB

So I request for you how can I get the DB value directly

The following context is the part of my LDAY_GEOM.geojson:

{"type":"Point","coordinates":[223495.9880411485,6757167.98900822,0.0]},"properties":{"IDRECEIVER":1,"HZ63":70.85,"HZ125":61.43,"HZ250":59.53,"HZ500":59.52,"HZ1000":60.93,"HZ2000":57.22,"HZ4000":51.38,"HZ8000":42.20,"LAEQ":64.37,"LEQ":72.37}},

**Reply by Nitnelav** [reply]
Hi @mshmoon,

"HZ" columns actually contain dB values. The proper indicator name would be "Leq,63Hz" for exemple. As it differs from the global values ("Leq" and "LAeq") it is named "HZ63" but its unit is indeed dB.

So for exemple the column "HZ500" contains Leq levels, in dB, for the frequency 500Hz.

**Reply by mshmoon** [reply]
> Hi @mshmoon,
> 
> "HZ" columns actually contain dB values. The proper indicator name would be "Leq,63Hz" for exemple. As it differs from the global values ("Leq" and "LAeq") it is named "HZ63" but its unit is indeed dB.
> 
> So for exemple the column "HZ500" contains Leq levels, in dB, for the frequency 500Hz.

Do you mean that I can directly use the value of HZ as decibel?
For example, my result is "HZ4000": 51.38. I can think it is 51.38 decibels obtained under the condition of HZ4000, right?

**Reply by Nitnelav** [reply]
Yes exactly, no conversion needed, if you want the noise level at 4000 Hz, you can use the "HZ4000" column directly.

**Reply by mshmoon** [reply]
> Yes exactly, no conversion needed, if you want the noise level at 4000 Hz, you can use the "HZ4000" column directly.

Thank you very much. You have answered my doubts and have been very helpful to me
